mlrun/utils/helpers.py

```diff
@@ -1295,3 +1295,25 @@ def ensure_git_branch(url: str, repo: git.Repo) -> str:
     if not branch and not reference:
         url = f"{url}#refs/heads/{repo.active_branch}"
     return url
+
+
+class DeprecationHelper(object):
+    """A helper class to deprecate old schemas"""
+
+    def __init__(self, new_target):
+        self._new_target = new_target
+
+    def _warn(self):
+        warnings.warn(
+            f"mlrun.api.schemas.{self._new_target.__name__} is deprecated. "
+            f"Please use mlrun.common.schemas.{self._new_target.__name__} instead.",
+            FutureWarning,
+        )
```

Comment:
I suggest passing a version as well so user will have an idea when we are planning to deprecate